contracts/implementation/PoolFactory.sol

```diff
@@ -213,28 +221,6 @@ contract PoolFactory is IPoolFactory, Ownable {
         return address(pairToken);
     }
 
-    /**
-     * @notice Sets the address of the associated `PoolKeeper` contract
-     * @param _poolKeeper Address of the `PoolKeeper`
-     * @dev Throws if provided address is null
-     * @dev Only callable by the owner
-     */
-    function setPoolKeeper(address _poolKeeper) external override onlyOwner {
-        require(_poolKeeper != address(0), "address cannot be null");
-        poolKeeper = IPoolKeeper(_poolKeeper);
-    }
-
-    /**
-     * @notice Sets the address of the associated `AutoClaim` contract
-     * @param _autoClaim Address of the `AutoClaim`
-     * @dev Throws if provided address is null
-     * @dev Only callable by the owner
-     */
-    function setAutoClaim(address _autoClaim) external override onlyOwner {
-        require(_autoClaim != address(0), "address cannot be null");
-        autoClaim = _autoClaim;
-    }
-
```

Comment:
I know this is still a WIP but I believe we want to keep these functions in, in case we want to start using a new `PoolKeeper` or `AutoClaim` contract